### ISSUE TITLE
Can the addition of low-quality images to the training dataset increase the network performance?

### ISSUE THREAD

**Issue by PROGRAMMINGENGINEER-NIKI** [AlexeyAB/darknet#5068]
I already trained the network with high-quality images (1920 by 1080 pixels) for a detection task. The result for mAP and IOU were 93% and 89% respectively.

I wanted to decrease the quality of my training data set using some available filters, then I use those low-quality images along with high-quality images to train the network again.

Does this method increase the accuracy (or, in general, performance) of the deep neural network (for a detection task)? Like mAP and IOU?

My data set is vehicle images.

**Comment by AlexeyAB** [comment]
> then I use those low-quality images along with high-quality images to train the network again.

If you want to Detect on low-quality images, then you should train with low-quality images.
